.evergreen/evergreen.yml.in

```diff
@@ -335,19 +335,29 @@ functions:
       params:
         file: tmp/expansions.yaml
         redacted: true
+    - command: shell.exec
+      params:
+        working_dir: src
+        shell: bash
+        script: |
+          export DISTRO_ID=${distro_id}
```

Comment:
```suggestion
```

Environment variables shouldn’t be preserved between evergreen commands, so this doesn’t do anything I think? If `DISTRO_ID` is otherwise not available in the command below, add it to its `env:` section like the other existing ones

Reply:
I did try to use env in the subshell, but it didn't work, it was expanding to empty, I think I was doing something wrong. I can try again and run the build only for one of the variants and see if it works.

Reply:
Yeah it worked, I think I forgot to generate the yaml before sending the patch.